pkg/runctl/logging.go

```diff
@@ -47,5 +48,32 @@ func getPipelineRunInfoForLogging(run *v1alpha1.PipelineRun) []interface{} {
 			"pipelineRunExecAuxNamespace", run.Status.AuxiliaryNamespace,
 		)
 	}
+	for loggingLabel, accessor := range customLoggingDetails {
```

Comment:
In a map the order is not guaranteed. But it would be desirable to keep the original order. Maybe a list (already in the configuration) is better.

Reply:
I'm not sure why the ordering should be important in the structured logging. You have key value pairs there.